text/trace/0157-messaging-semantic-conventions.md

```diff
@@ -0,0 +1,174 @@
+# Scenarios for Tracing semantic conventions for messaging
+
+This document aims to capture scenarios and a road map, both of which will
+serve as a basis for [stabilizing](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md#stable)
+the [existing semantic conventions for messaging](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/messaging.md),
+which are currently in an [experimental](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md#experimental)
+state. The goal is to declare messaging semantic conventions stable before the
+end of 2021.
+
+## Motivation
+
+While a variety of different messaging systems exist, most of them have similar
+concepts, and standardization efforts exist. Messaging systems play an
+important part in many observability scenarios, and OpenTelemetry must provide
```

Comment:
Wouldn't it be better if this is expressed somewhat differently: i.e. many scenarios involve using messaging systems, and for distributed tracing to be useful across the entire scenario, having good observability for messaging operations is critical.